### PR TITLE
Fix ocp4-workload-ccn-stable to replace user.info module

### PR DESCRIPTION
**Pull request by danieloh30**
##### SUMMARY

Fix ocp4-workload-ccn-stable to replace user.info module

##### ISSUE TYPE

- Bugfix Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: ansible/roles/ocp4-workload-ccnrd-stable/tasks/post_workload.yml [hunk 1]
```diff
@@ -1,78 +1,72 @@
...
-    - "user.info: Module 1 http://guides-m1-labs-infra.{{ route_subdomain }}"
+      Module 1 http://guides-m1-labs-infra.{{ route_subdomain }}
 
 - name: output workshop info guides
   when: ("m2" in modules)
-  debug:
+  agnosticd_user_info:
     msg:
-    - "user.info: Module 2 http://guides-m2-labs-infra.{{ route_subdomain }}"
+      Module 2 http://guides-m2-labs-infra.{{ route_subdomain }}
```
Comment (jkupferer) [inline ansible/roles/ocp4-workload-ccnrd-stable/tasks/post_workload.yml:38]: Did you mean to use a yaml block quote here for the `msg:` line?

File: ansible/roles/ocp4-workload-ccnrd-stable/tasks/post_workload.yml [hunk 2]
```diff
@@ -1,78 +1,72 @@
...
 
 - name: output workshop info guides
   when: ("m3" in modules)
-  debug:
+  agnosticd_user_info:
     msg:
-    - "user.info: Module 3 http://guides-m3-labs-infra.{{ route_subdomain }}"
+      Module 3 http://guides-m3-labs-infra.{{ route_subdomain }}
```
Comment (jkupferer) [inline ansible/roles/ocp4-workload-ccnrd-stable/tasks/post_workload.yml:44]: Did you mean to use a yaml block quote here for the `msg:` line?

File: ansible/roles/ocp4-workload-ccnrd-stable/tasks/post_workload.yml [hunk 3]
```diff
@@ -1,78 +1,72 @@
...
 
 - name: output workshop info guides
   when: ("m4" in modules)
-  debug:
+  agnosticd_user_info:
     msg:
-    - "user.info: Module 4 http://guides-m4-labs-infra.{{ route_subdomain }}"
+      Module 4 http://guides-m4-labs-infra.{{ route_subdomain }}
```
Comment (jkupferer) [inline ansible/roles/ocp4-workload-ccnrd-stable/tasks/post_workload.yml:50]: Did you mean to use a yaml block quote here for the `msg:` line?

File: ansible/roles/ocp4-workload-ccnrd-stable/tasks/post_workload.yml [hunk 4]
```diff
@@ -1,78 +1,72 @@
...
+      NOTE: Workspaces in CodeReady AND service mesh are provisioned asynchronously and may not be accessible until rollout finishes shortly.
   when: not silent|bool
 
 # Leave this as the last task in the playbook.
 - name: post_workload tasks complete
-  debug:
-    msg: "Post-Workload Tasks completed successfully."
+  agnosticd_user_info:
+    msg: 
+      Post-Workload Tasks completed successfully.
```
Comment (jkupferer) [inline ansible/roles/ocp4-workload-ccnrd-stable/tasks/post_workload.yml:71]: Did you mean to use a yaml block quote here for the `msg:` line?

File: ansible/roles/ocp4-workload-ccnrd-stable/tasks/post_workload.yml (hunk 2 above)
Comment (danieloh30) [reply ansible/roles/ocp4-workload-ccnrd-stable/tasks/post_workload.yml:44]: Updated with a pipe

File: ansible/roles/ocp4-workload-ccnrd-stable/tasks/post_workload.yml (hunk 1 above)
Comment (danieloh30) [reply ansible/roles/ocp4-workload-ccnrd-stable/tasks/post_workload.yml:38]: Updated with a pipe

File: ansible/roles/ocp4-workload-ccnrd-stable/tasks/post_workload.yml (hunk 2 above)
Comment (danieloh30) [reply ansible/roles/ocp4-workload-ccnrd-stable/tasks/post_workload.yml:44]: Updated with a pipe

File: ansible/roles/ocp4-workload-ccnrd-stable/tasks/post_workload.yml (hunk 3 above)
Comment (danieloh30) [reply ansible/roles/ocp4-workload-ccnrd-stable/tasks/post_workload.yml:50]: Updated with a pipe

File: ansible/roles/ocp4-workload-ccnrd-stable/tasks/post_workload.yml (hunk 4 above)
Comment (danieloh30) [reply ansible/roles/ocp4-workload-ccnrd-stable/tasks/post_workload.yml:71]: Updated with a pipe

**Review by jkupferer — APPROVED** [review]

**Pull request merged**